components/google-cloud/google_cloud_componets/mb_sdk_component/utils.py

```diff
@@ -0,0 +1,98 @@
+# Lint as: python3
+# Copyright 2019 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module for creating pipeline component definitions based on Model Builder SDK."""
```

Comment:
What's the released named of this SDK?

Reply:
it is Cloud AI Platform SDK changing the comment accordingly.